bigbluebutton-html5/imports/api/users/server/handlers/presenterAssigned.js

```diff
@@ -1,10 +1,13 @@
 import Users from '/imports/api/users';
+import PresentationPods from '/imports/api/presentation-pods';
 import changeRole from '/imports/api/users/server/modifiers/changeRole';
+import assignPresenter from '../methods/assignPresenter';
 
-export default function handlePresenterAssigned({ body }, meetingId) {
+export default function handlePresenterAssigned(credentials, meetingId) {
```

Comment:
Do we need to introduce `credentials`? `{ body }` seems to be enough...
This is regarding

```
- export default function handlePresenterAssigned({ body }, meetingId) {
+ export default function handlePresenterAssigned(credentials, meetingId) {

+  const { body } = credentials;
```